internal/cmd/logout_test.go

```diff
@@ -316,4 +331,12 @@ func TestLogout(t *testing.T) {
 		assert.ErrorContains(t, err, `"infra logout" accepts at most 1 argument`)
 		assert.ErrorContains(t, err, `Usage:  infra logout [SERVER]`)
 	})
+
+	t.Run("expired session fails", func(t *testing.T) {
```

Comment:
this should check if the expired config is removed

Reply:
done (at least I think that is what you meant for me to write up, the equateEmpty() parts).